docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -490,3 +492,251 @@ projects.
 **When do the changes take effect?**
 
 In Kotlin 1.9.0, a deprecation warning is introduced when the `android` name is used in Kotlin Multiplatform projects.
+
+<anchor name="declaring-multiple-targets"></anchor>
+## Declaring multiple targets of the same kind
+
+**What's changed?**
+
+We discourage declaring multiple targets of the same kind in a single Gradle project.
+
+Example:
+```kotlin
+kotlin {
+    jvm("jvmKtor")
+    jvm("jvmOkio") // is not recommended and will produce deprecation warning
+}
+```
+
+One popular case is having two related pieces of code together. For example, you might be compelled to use
+`jvm("jvmKtor")` and `jvm("jvmOkio")` in your `:shared` Gradle project for providing an implementation of networking
+code based on Ktor or OKIO:
```

Comment:
FWIW, the example might be better with `Ktor` & `OkHttp`. Okio is the low level IO library but doesn't deal with networking.

Reply:
Indeed, that was a brainlag from me (though about `OkHttp` but typed `Okio`). Thanks for pointing out, Martin! 👍 

cc @danil-pavlov could you fix that during the TW editing?